tools/appliance/systemvmtemplate/template.json

```diff
@@ -0,0 +1,82 @@
+{
+  "_license": "Apache License 2.0",
```

Comment:
is this passing the rat like this?

Reply:
No, just for information. Unfortunately, the packer jsons cannot have any license header but they allow custom tags/comments like this. They've been all removed from RAT checks.